[ADD] stock_landed_costs_security: New user group to manage landed costs (#1017)
OCA/stock-logistics-workflow: stock_landed_costs_security/readme/DESCRIPTION.rst

> This addons adds the Landing cost user group. Any user mapped to this grou can create an validate landing costs.

Reviewer: ```suggestion
This addon adds the "Landing cost" user group. Any user mapped to this group can create and validate landing costs.
```

> This addon adds the "Landing cost" user group. Any user mapped to this group can create and validate landing costs.

Reviewer: ```suggestion
This module promotes the "Landing cost" feature to a full application, with its icon and specific menu, accessible through an specific user group. Any user mapped to this group can create and validate landing costs without extra permissions.
```
